Library/Formula/zurl.rb

```diff
@@ -0,0 +1,21 @@
+class Zurl < Formula
+  homepage "https://github.com/fanout/zurl"
+  url "http://packages.fanout.io/source/zurl-1.3.1.tar.bz2"
+  sha1 "8a04ad092a4fd9b174a352b910c190466fc39eb9"
+
+  depends_on "pkg-config" => :build
+  depends_on "curl"
+  depends_on "qt"
+  depends_on "zeromq"
+  depends_on "qjson"
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/zurl", "--version"
```

Comment:
The problem is zurl is a daemon and communicates with remote HTTP servers so it's a bit hard to test in a simple script. Let me see if I can adapt the test from curl.rb. The script could launch zurl, fetch a file through it, and then kill zurl afterwards.